kroxylicious-api/src/main/java/io/kroxylicious/proxy/package-info.java

```diff
@@ -38,4 +38,5 @@
  * the communication between the client and broker.</p>
  *
  */
+@ApiMaturity.Unstable
```

Comment:
Just mark everything as unstable for now.

Reply:
HAHAHA, I really thought you were pushing for at least evolving for filter API.

Reply:
Just getting the mechanism for evolution in place first. Actually changing the annotations would be a whole other  discussion.